### PR TITLE
added dts-boot for tests

### PR DESCRIPTION
**Pull request by WiktorG351**
This PR adds the version of `dts-boot` which creates the DES keys, which is necessary for tests on QEMU.

### CODE REVIEW

File: unit-tests/README.md [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+We need credentials for each test variant. You can use provided template and
+fill it in accordingly.
+
+```bash
+cp des-credentials.sh.example des-credentials.sh
+```
+
+### Running automatically
+
+Some scenarios are have been already migrated into [OSFV](TBD).
```
Comment (m-iwanicki) [inline unit-tests/README.md:26]: Currently this 'link' doesn't lead anywhere. Should it stay that way?

File: unit-tests/dts-boot [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  fi
+elif [ $TEST_DES != "y" ] && [ -f "${CLOUD_PASS_FILE}" ]; then
+  rm -f "${CLOUD_PASS_FILE}"
+fi
+
+if [ -f /tmp/bios.bin ]; then
+  rm /tmp/bios.bin
+fi
+
+wget https://dl.3mdeb.com/open-source-firmware/Dasharo/novacustom_nv4x_adl/v1.7.2/novacustom_nv4x_adl_v1.7.2.rom -O /tmp/bios.bin
```
Comment (m-iwanicki) [inline unit-tests/dts-boot:44]: Why is this bios hardcoded? Does it matter which one is downloaded or not?

**Comment by macpijan** [comment]
@WiktorG351 Shouldn't we allow for overriding **some** of the veriables? I have closed this: https://github.com/Dasharo/dts-scripts/pull/9 as it was too much. In meta-dts I have selected just a few.

File: unit-tests/README.md [hunk 3]
```diff
@@ -0,0 +1,268 @@
+# DTS scripts unit tests
```
Comment (macpijan) [inline unit-tests/README.md:1]: I know it was named this way iniitally, but these are not unit tests really.

How about we at lest rename this to simply `tests` ?

File: unit-tests/README.md (hunk 1 above)
Comment (macpijan) [reply unit-tests/README.md:26]: I think I meant to use this MR back then: https://github.com/Dasharo/open-source-firmware-validation/pull/333

**Comment by macpijan** [comment]
Integrated here: https://github.com/Dasharo/dts-scripts/pull/23

**Pull request closed without merging**